db/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+    //    application
+    kotlin("jvm")
+    id("org.flywaydb.flyway") version ("5.1.3")
+}
+
+dependencies {
+    compile(kotlin("stdlib", embeddedKotlinVersion))
+    compile("org.jetbrains.kotlinx", "kotlinx-coroutines-core", "0.22.5")
```

Comment:
Is this dependency necessary?

Reply:
removed

Reply:
Another reply to an another comment.
Regarding including flyway logic into `build.gradle.kts`.
I've created another file do make flyway logic separated, easily replaced with different config.
DB scripts now are isolated, could be extracted as separate project if needed.

So, different file allows us to do not mix up all build logic in one file, keeping scripts readable.

About making 'build' dependent on'flywayMigrate' task - could be good idea, but let's discuss on-site, I have doubts here.

Reply:
+1 for enabling Flyway execution out of Docker. In the current setup, if Docker host is (temporarily) not available in an environment and the container cannot be run locally (Windows 7), Flyway cannot be run out-of-the-box.